package.json

```diff
@@ -65,6 +65,7 @@
     "mongoose": "^5.3.2",
     "mongoose-unique-validator": "^2.0.2",
     "mysql": "^2.13.0",
+    "node-fetch": "^2.3.0",
```

Comment:
I believe in this project we use `http`: https://nodejs.org/api/http.html#http_http_request_options_callback

@simonLouvet : Can you confirm?

Reply:
I believe the `http/https` libraries (like the `request` library) are faily low-level. When you deal with more complex situations, for example if you are behind a proxy, if you need to handle CORS or redirections, it will require tweaks. (I've seen that for some components, the `follow-redirects` library is used just to handle the redirections.)
The advantage of `node-fetch` is that it pretty much handles all cases.

Reply:
@grizio I believe the `http/https` libraries (like the `request` library) are faily low-level. When you deal with more complex situations, for example if you are behind a proxy, if you need to handle CORS or redirections, it will require tweaks. (I've seen that for some components, the `follow-redirects` library is used just to handle the redirections.)
The advantage of `node-fetch` is that it pretty much handles all cases.